app/models/roster/creator.rb

```diff
@@ -57,12 +57,15 @@ def initialize(organization:, identifier_name:, **options)
 
     # Internal: Create create a Roster.
     #
+    # rubocop:disable Metrics/MethodLength
     def perform
       ensure_organization_does_not_have_roster!
 
       ActiveRecord::Base.transaction do
         @roster = Roster.new(identifier_name: @identifier_name)
-        add_identifiers_to_roster(@options[:identifiers]) if @options.key?(:identifiers)
+
+        google_ids = @options[:google_user_ids].nil? ? [] : @options[:google_user_ids]
```

Comment:
Optional `:google_user_ids` key is used to associate Google Classroom student ID with the Roster Entry